docs/workers.md

```diff
@@ -205,7 +205,7 @@ information.
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/context/.*$
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/members$
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/state$
-    ^/_matrix/client/(v1|unstable/org.matrix.msc2946)/rooms/.*/hierarchy$
+    ^/_matrix/client/v1/rooms/.*/hierarchy$
```

Comment:
Shouldn't this be `v3`? Surely its not `v1`?

Reply:
It is a new endpoint, so it was added as v1. The only endpoints that became v3 were the ones which existed as r0.

See https://github.com/matrix-org/synapse/blob/d4713d3e335b21d12284ddd8ebd00e38abcfd521/synapse/rest/client/room.py#L1196-L1201

Reply:
Hmm, I forgot that that was how we decided to do it

Reply:
It seems to confuse a lot of people. 😄